src/opamPrinter.ml

```diff
@@ -324,3 +324,340 @@ module Preserved = struct
     items txt orig.file_contents f.file_contents
```

Comment:
GitHub won't let me note this on the correct line, but the line above wants to be `let[@ocaml.warning "-3"] =`